### PR TITLE
Metrics - Don't exclaim unless > 1

### PR DESCRIPTION
**Pull request by paragonie-scott**
"You have validated 0 transactions!" sounds a little less enthusiastic that intended. This uses a period instead.

After patch, it also says "a transaction".

### CODE REVIEW

**Comment by daira** [comment]
utACK.

File: src/metrics.cpp [hunk 1]
```diff
@@ -94,6 +94,7 @@ int printMetrics(size_t cols, int64_t nStart, bool mining)
     int hours = (uptime - (days * 24 * 60 * 60)) / (60 * 60);
     int minutes = (uptime - (((days * 24) + hours) * 60 * 60)) / 60;
     int seconds = uptime - (((((days * 24) + hours) * 60) + minutes) * 60);
+    int validatedCount = 0;
```
Comment (str4d) [inline src/metrics.cpp]: I'd personally prefer defining this where it is used.

File: src/metrics.cpp [hunk 2]
```diff
@@ -110,7 +111,14 @@ int printMetrics(size_t cols, int64_t nStart, bool mining)
     std::cout << strDuration << std::endl;
     lines += (strDuration.size() / cols);
 
-    std::cout << "- " << strprintf(_("You have validated %d transactions!"), transactionsValidated.get()) << std::endl;
+    validatedCount = transactionsValidated.get();
```
Comment (str4d) [inline src/metrics.cpp]: ie. here.

File: src/metrics.cpp [hunk 3]
```diff
@@ -110,7 +111,14 @@ int printMetrics(size_t cols, int64_t nStart, bool mining)
...
+    if (validatedCount > 1) {
+      std::cout << "- " << strprintf(_("You have validated %d transactions!"), validatedCount) << std::endl;
+    } else if (validatedCount == 1) {
+      std::cout << "- " << strprintf(_("You have validated %d transaction."), validatedCount) << std::endl;
```
Comment (str4d) [inline src/metrics.cpp]: Why is this not an exclamation?

**Comment by str4d** [comment]
I've force-pushed to tweak the existing commit message, and have addressed my comments.

**Comment by str4d** [comment]
ACK.

@zkbot r+

**Pull request merged**